LDAR_Sim/install/requirements.txt

```diff
@@ -12,3 +12,6 @@ plotnine==0.7.1
 ephem
 gdal==3.2.1
 boto3 == 1.16.63
+scipy
+yaml
+scikit-learn
```

Comment:
Can we pin the versions of scipy and scikit-learn?  I'll update the pipfile and run a pipfile lock so both requirements and pipfiles are up to date.

Just suggesting pinning these versions as these packages and numpy closely relate.

Reply:
Fixed and verified with Conda